carsons/carsons.py

```diff
@@ -177,6 +195,10 @@ def compute_D(self, i, j) -> float:
 
         return self.calculate_distance(self.phase_positions[i], (xⱼ, -yⱼ))
 
+    def compute_C(self, i, j) -> float:
```

Comment:
it seems like this may not be called? do we need it?

Reply:
Nope...